tests/test_frompyobject.rs

```diff
@@ -6,6 +6,21 @@ use pyo3::PyMappingProtocol;
 #[macro_use]
 mod common;
 
+/// Helper function that concatenates the error message from
+/// each error in the traceback into a single string that can
+/// be tested.
+fn extract_traceback(mut error: PyErr) -> String {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+    let mut error_msg = error.to_string();
+    while let Some(err) = error.cause(py) {
+        error_msg.push_str(": ");
+        error_msg.push_str(&err.to_string());
+        error = err;
```

Comment:
Nit: would be nice to use `cause` for the binding:

```suggestion
    while let Some(cause) = error.cause(py) {
        error_msg.push_str(": ");
        error_msg.push_str(&cause.to_string());
        error = cause;
```

Reply:
:+1: